src/app/api/auth/token/route.ts

```diff
@@ -42,6 +27,23 @@ async function requestToken(code: string) {
             return data.access_token;
         }
     } catch (error) {
-        return new Response('Unauthorized', { status: 401 });
+        return 'error';
```

Comment:
Should define this as a constant so if you need to update it you will not to rewrite every string.

Reply:
defined this inside the constants file, changed all return messages inside each route.ts